pkg/bmc/redfish.go

```diff
@@ -19,10 +20,21 @@ func newRedfishAccessDetails(parsedURL *url.URL) (AccessDetails, error) {
 	}, nil
 }
 
+func newRedfishVirtualMediaAccessDetails(parsedURL *url.URL) (AccessDetails, error) {
+	bmcType := strings.Replace(parsedURL.Scheme, "+virtualmedia", "", 1)
+	return &redfishAccessDetails{
+		bmcType:        bmcType,
```

Comment:
The Type() method (which returns bmcType) should return the actual URL scheme, not a mangled version.
This would require changes to DriverInfo(), though not if you changed `+virtualmedia` to `-virtualmedia`.

Reply:
Addressed in #357